compatible.csv

```diff
@@ -1,5 +1,6 @@
-CKB,0.110,0.109,0.108,0.107,0.106,0.105,0.104,0.103
-Neuron,,,,,,,,
-0.110,yes,yes,no,no,no,no,no,no
-0.106,no,no,yes,yes,yes,yes,no,no
-0.103,no,no,no,no,no,no,yes,yes
+CKB,0.111,0.110,0.109,0.108,0.107,0.106,0.105,0.104,0.103
+Neuron,,,,,,,,,
+0.111,yes,yes,yes,no,no,no,no,no,no
```

Comment:
Not really related to this PR, but I'd like to ask the source of this table, it seems like it's not the same as the compatibility table from [CKB releases](https://github.com/nervosnetwork/ckb/releases/tag/v0.111.0)

Reply:
The 109 version changes an RPC function name(`get_fee_rate_statics` -> `get_fee_rate_statistics`), and the RPC function that the Neuron needs. So we set it to false with the 108 version.
https://github.com/nervosnetwork/ckb/releases/tag/v0.109.0